WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -35,11 +35,7 @@ final class OrderListViewController: UIViewController, GhostableViewController {
 
     /// Main TableView.
     ///
-    private lazy var tableView: UITableView = {
-        let tableView = UITableView(frame: .zero, style: .grouped)
-        tableView.accessibilityIdentifier = "orders-table-view"
```

Comment:
nit: we tend to declare all the things related to a specific view inside the code, and not on in the `xib` or `storyboard` files. I saw that you added the  `accessibilityIdentifier` in the xib file, and it's easy to miss it or declare it a second time.

Reply:
Thanks for letting me know :) I changed it accordingly